setup/pattoo_installation.py

```diff
@@ -359,7 +358,7 @@ def main():
         from pattoo_shared.installation import packages, systemd, environment
 
         # Setup virtual environment
-        if getpass.getuser() != 'travis':
+        if getpass.getuser() == 'root':
             pattoo_home = get_pattoo_home()
```

Comment:
Is this the home of the `pattoo` user in this case? It should be based on our operating assumptions right?

Reply:
Yup it assumes  the home directory of the pattoo user to be `/home/pattoo` if the user isn't created as yet